### DISCUSSION TITLE
8-0 Part 3 Crafting local maps

### DISCUSSION THREAD

**Discussion by aaronbirr** [ua-gist604b-f24/syllabus#47] [General]
Anyone run into this problem:
Trying to run the **cd CONUS_100m_NE_LC_SR_W.zip** from below and get the following error

wget http://www.shadedrelief.com/NE_100m/CONUS_100m_NE_LC_SR_W.zip
unzip CONUS_100m_NE_LC_SR_W.zip
cd CONUS_100m_NE_LC_SR_W.zip

<img width="925" alt="Screenshot 2024-12-02 173030" src="https://github.com/user-attachments/assets/7e6258ec-2a03-4631-a68c-525c168f05d8">

**Comment by travisz09** [comment]
Use dir to check the directories, from your current directory, and make sure there are no typos. Also, I may be wrong, but you probably want to access the unzipped directory, not the zipped.

**Reply by aaronbirr** [reply]
Thank you! Removing the .zip worked - cd CONUS_100m_NE_LC_SR_W
